vignettes/programming_strategy.Rmd

```diff
@@ -645,7 +643,7 @@ fun_xxx <- function(dataset, some_param, other_param) {
     what = "fun_xxx()", 
```

Comment:
Could you style the call?
![image](https://github.com/user-attachments/assets/82bea046-4c2d-44bd-b571-8fc363529402)

Reply:
styled!